src/core/client/admin/components/ModerateCard/ModerateCardDetailsContainer.css

```diff
@@ -0,0 +1,4 @@
+.button {
+  text-transform: uppercase;  
+  font-size: 14px;
```

Comment:
We have a variable for this `var(--font-size-2)`

I'm going to approve this though, as I'll have to edit it on the branding refactor pass anyways.